projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.material.end_rod": "末地烛",
+    "tetra.material.end_rod.prefix": "末地烛",
+    "tetra.material.oak": "木",
+    "tetra.material.oak.prefix": "木制",
+    "tetra.material.warped": "诡异",
+    "tetra.material.warped.prefix": "诡异",
+    "tetra.material.wool": "羊毛",
+    "tetra.material.wool.prefix": "羊毛",
+    "tetra.material.blackstone": "黑石",
+    "tetra.material.blackstone.prefix": "黑石",
+    "tetra.material.netherite": "下界合金",
+    "tetra.material.netherite.prefix": "下界合金",
+    "tetra.material.dark_oak": "深色橡木",
+    "tetra.material.dark_oak.prefix": "深色橡木",
+    "tetra.material.emerald": "绿宝石",
+    "tetra.material.emerald.prefix": "绿宝石",
+    "tetra.material.hide": "兔子皮",
+    "tetra.material.hide.prefix": "兔子皮",
+    "tetra.material.twisting_vine": "缠怨藤",
+    "tetra.material.twisting_vine.prefix": "缠怨藤",
+    "tetra.material.forged_beam": "再利用梁杆",
+    "tetra.material.forged_beam.prefix": "再利用梁杆",
+    "tetra.material.vine": "藤蔓",
+    "tetra.material.vine.prefix": "藤蔓",
+    "tetra.material.spruce": "云杉",
+    "tetra.material.spruce.prefix": "云杉",
+    "tetra.material.crimson": "绯红",
+    "tetra.material.crimson.prefix": "绯红",
+    "tetra.material.bronze": "青铜",
+    "tetra.material.bronze.prefix": "青铜",
+    "tetra.material.copper": "铜",
+    "tetra.material.copper.prefix": "铜",
+    "tetra.material.electrum": "琥珀金",
+    "tetra.material.electrum.prefix": "琥珀金",
+    "tetra.material.lead": "铅",
+    "tetra.material.lead.prefix": "铅",
+    "tetra.material.nickel": "镍",
+    "tetra.material.nickel.prefix": "镍",
+    "tetra.material.silver": "银",
+    "tetra.material.silver.prefix": "银",
+    "tetra.material.steel": "钢",
+    "tetra.material.steel.prefix": "钢",
+    "tetra.material.tin": "锡",
+    "tetra.material.tin.prefix": "锡",
+    "tetra.material.baobab": "猴面包木",
+    "tetra.material.baobab.prefix": "猴面包木",
+    "tetra.material.cherry": "樱木",
+    "tetra.material.cherry.prefix": "樱木",
+    "tetra.material.maple": "枫木",
+    "tetra.material.maple.prefix": "枫木",
+    "tetra.material.agate": "玛瑙",
+    "tetra.material.agate.prefix": "玛瑙",
+    "tetra.material.amethyst": "紫水晶",
+    "tetra.material.amethyst.prefix": "紫水晶",
+    "tetra.material.certus_quartz": "赛特斯石英",
+    "tetra.material.certus_quartz.prefix": "赛特斯石英",
+    "tetra.material.citrine": "黄水晶",
+    "tetra.material.citrine.prefix": "黄水晶",
+    "tetra.material.ruby": "红宝石",
+    "tetra.material.ruby.prefix": "红宝石",
+    "tetra.material.sapphire": "蓝宝石",
+    "tetra.material.sapphire.prefix": "蓝宝石",
+    "tetra.material.serendibite": "盐硅硼钙石",
+    "tetra.material.serendibite.prefix": "盐硅硼钙石",
+    "tetra.material.hemp": "大麻",
+    "tetra.material.hemp.prefix": "大麻",
+    "tetra.material.osmium": "锇",
+    "tetra.material.osmium.prefix": "锇",
+    "tetra.material.treated_wood": "防腐木",
+    "tetra.material.treated_wood.prefix": "防腐木",
+    "tetra.material.dragon_bone": "龙骨",
+    "tetra.material.dragon_bone.prefix": "龙骨",
+    "tetra.material.wither_bone": "凋零之骨",
```

|Feedback Priority|
|---|
|⛰ Mountain / **You must make this correct**|

```suggestion
    "tetra.material.wither_bone": "凋灵之骨",
```